zephyr/wrapper.c

```diff
@@ -528,6 +537,10 @@ int task_main_start(struct sof *sof)
 	/* init pipeline position offsets */
 	pipeline_posn_init(sof);
 
+	(void)notifier_register(NULL, scheduler_get_data(SOF_SCHEDULE_LL_TIMER),
```

Comment:
Can we have a comment here stating the long term solution so everyone knows this will be temporary. This can be another PR, feel free to create a feature for this too.

Reply:
Submitted in PR #4441